modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts

```diff
@@ -13,13 +13,13 @@ import * as Icons from '../icons/Icons';
 
 const english: Record<string, string> = {
   'colorcustom.rgb.red.label': 'R',
-  'colorcustom.rgb.red.description': 'Red component',
+  'colorcustom.rgb.red.description': 'Red component (R)',
```

Comment:
Remember to PR this in the POT files add a ticket for that or separate PR.

Reply:
PR now please.

Reply:
https://github.com/tinymce/tinymce-premium/pull/344